index.html

```diff
@@ -379,7 +380,7 @@ <h2>Glossary</h2>
 <a href="#links"><abbr title="Back to Alphabetical links">&#x2191;</abbr></a>
 
 <div class="letter_anchor" id="o">O</div>
-<p><dfn class="lint-ignore export">Othographic syllable</dfn> A [=typographic character unit=] that includes more than one [=grapheme cluster=]. These are most commonly found in Brahmi-derived scripts, such as Devanagari, or Balinese, when forming conjuncts or stacks. This term is used but not defined in the Unicode Standard: it is defined here for use by W3C documents, such as layout requirements.</p>
+<p><dfn class="lint-ignore export">Othographic syllable</dfn> A [=typographic character unit=] that includes one or more [=grapheme clusters=]. These are most commonly found in Brahmi-derived scripts, such as Devanagari, or Balinese, when forming conjuncts or stacks. They commonly demarcate sequences of characters that are different from phonetic syllables, and they may even span word boundaries. See also the &lt;a href=&quot;https://unicode.org/glossary/#orthographic_syllable&quot; target=&quot;_blank&quot;&gt;definition&lt;/a&gt; in the Unicode Standard.</p>
```

Comment:
Note that you've got `&lt;a...` instead of `<a...`, so it appears as markup in https://pr-preview.s3.amazonaws.com/w3c/i18n-glossary/pull/77.html#dfn-othographic-syllable.

Reply:
That's wierd. Thanks for noticing. Fixed.